templates/cassiopeia/html/layouts/chromes/cardGrey.php

```diff
@@ -21,34 +21,22 @@
 }
 
 $moduleTag              = $params->get('module_tag', 'div');
-$moduleAttribs          = [];
+$modulePos 		= $module->position;
```

Comment:
```suggestion
$modulePos              = $module->position;
```
Again there are tabs used instead of spaces. Maybe my mistake before. Correcting to space.